src/SDKs/DataMigration/DataMigration.Tests/Helpers/Utilities.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.DataMigration;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
+using Microsoft.Azure.Test.HttpRecorder;
+using Microsoft.Rest;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Net.Http;
+using System.Threading;
+
+namespace DataMigration.Tests.Helpers
+{
+    public static class Utilities
+    {
+        /// <summary>
+        /// Random generator.
+        /// </summary>
+        private static Random random = new Random();
+
+        public static bool IsTestTenant = false;
+        private static HttpClientHandler Handler = null;
+
+        // These should be filled in only if test tenant is true
+        public static string certName = null;
+        public static string certPassword = null;
+        private static string testSubscription = null;
+        private static Uri testUri = null;
+
+        // These are used to create default accounts
+        public static string DefaultLocation = IsTestTenant ? null : "westus2";
+
+        public static ResourceManagementClient GetResourceManagementClient(MockContext context, RecordedDelegatingHandler handler)
+        {
+            if (IsTestTenant)
+            {
+                return null;
+            }
+            else
+            {
+                handler.IsPassThrough = true;
+                ResourceManagementClient resourcesClient = context.GetServiceClient<ResourceManagementClient>(handlers: handler);
+                return resourcesClient;
+            }
+        }
+
+        public static DataMigrationServiceClient GetDataMigrationManagementClient(MockContext context, RecordedDelegatingHandler handler)
+        {
+            DataMigrationServiceClient dmClient;
+            if (IsTestTenant)
+            {
+                dmClient = new DataMigrationServiceClient(new TokenCredentials("xyz"), GetHandler());
+                dmClient.SubscriptionId = testSubscription;
+                dmClient.BaseUri = testUri;
+            }
+            else
+            {
+                handler.IsPassThrough = true;
+                dmClient = context.GetServiceClient<DataMigrationServiceClient>(handlers: handler);
+            }
+            return dmClient;
+        }
+
+        private static HttpClientHandler GetHandler()
+        {
+#if DNX451
```

Comment:
@hitenjava we do not support this target? These projects are no longer DNX based projects.
We only support one compile constant and that is FullNetFx

Reply:
Done.